ynput/OpenPype
# fusion: removing hardcoded template name for saver

## Changelog Description
Fusion is not hardcoded for `render` anatomy template only anymore. This was blocking AYON deployment.

## Testing notes:
1. open fusion in your testing project in ayon with default anatomy tempates which are not providing `render` template.
2. create render saver and see that all is created as expected.
